### PR TITLE
Remove 1 million cap limit in the max field for the product ID

### PR DESCRIPTION
**Pull request by marsaldev**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove the 1 million (front end) limitation to search IDs greater than 1 million for the product ID field
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | Open the product catalog list and type 1000001 in the max text field for the product ID. Without the PR, you cannot do it. The PR allows it.
| Possible impacts? | no

### CODE REVIEW

**Comment by matks** [comment]
@marsaldev if PR https://github.com/PrestaShop/PrestaShop/pull/30351 is merged inside 8.0.x ; this bugfix will be backported from 8.0.x to develop by a merge

So I think this PR is maybe not useful

**Comment by marsaldev** [comment]
@matks @jolelievre 
I've closed the other PR after a discussion with Maintainers/Committers, let's keep this one as the good one. 
Thanks 😊

**Comment by paulnoelcholot** [comment]
Hello @marsaldev,

I have tested the PR and this PR didn't corrige all the issues. It's not possible to insert "1000001" in Price max field and in Quantity max field.

There is also a problem with the Price min and Quantity min fields, because "1000001" becomes "1000000".

https://watch.screencastify.com/v/EyjCIlKXD6Is40fWF0cA

**Comment by marsaldev** [comment]
Hi @paulnoelcholot, I've never talked about "Price" field, but product ID field 😊

**Comment by marsaldev** [comment]
I've update the description, maybe it was confusing because I wrote it only in the title 😊

**Comment by paulnoelcholot** [comment]
My apologies ! And thanks you :)

**Comment by marsaldev** [comment]
@paulnoelcholot no worries at all! 😉

**Review by lartist — APPROVED** [review]

**Review by mflasquin — DISMISSED** [review]
Maybe you can also remove the limitation for the fields price and quantity to fix this ticket ? https://github.com/PrestaShop/PrestaShop/issues/30374

**Comment by marsaldev** [comment]
@mflasquin 
No problem for me 😉 

@MatShir what do you think? Could be helpful to remove this limit for price and quantity too?

I think it's hard to find a shop that sells products for 1million euros (but maybe could happen in another value?), or to search some products that have more than 1 million stock in the warehouse 🤔

**Comment by marsaldev** [comment]
@mflasquin @matks 
Can we unblock this situation? 😊

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Ping @MatShir @mflasquin - I think we could keep the original scope of the PR and move with this.

**Comment by MatShir** [comment]
Let's merge as it is now ! We can make another PR for the other issues. They are legit bugs to fixed

**Comment by paulnoelcholot** [comment]
Hello @marsaldev,

I have tested the PR and it's OK for me ! :tada: 

Thanks!

**Review by paulnoelcholot — APPROVED** [review]

**Comment by kpodemski** [comment]
thank you @marsaldev :)

**Pull request merged**
